skins/Chameleon/SkinChameleon.php

```diff
@@ -0,0 +1,555 @@
+
+        // Build additional attributes for navigation urls
+        $nav = $this->data['content_navigation'];
+
+        if ($wgVectorUseIconWatch) {
+            $mode = $this->getSkin()->getTitle()->userIsWatching() ? 'unwatch' : 'watch';
+            if (isset( $nav['actions'][$mode] )) {
+                $nav['views'][$mode] = $nav['actions'][$mode];
+                $nav['views'][$mode]['class'] = rtrim( 'icon ' . $nav['views'][$mode]['class'], ' ' );
+                $nav['views'][$mode]['primary'] = true;
+                unset( $nav['actions'][$mode] );
+            }
+        }
+
+        $xmlID = '';
+        foreach ($nav as $section => $links) {
+            foreach ($links as $key => $link) {
+                if ($section == 'views' && !( isset( $link['primary'] ) && $link['primary'] )) {
+                    $link['class'] = rtrim( 'collapsible ' . $link['class'], ' ' );
+                }
+
+                $xmlID = isset( $link['id'] ) ? $link['id'] : 'ca-' . $xmlID;
+                $nav[$section][$key]['attributes'] =
+                    ' id="' . Sanitizer::escapeId( $xmlID ) . '"';
+                if ($link['class']) {
+                    $nav[$section][$key]['attributes'] .=
+                        ' class="' . htmlspecialchars( $link['class'] ) . '"';
+                    unset( $nav[$section][$key]['class'] );
+                }
+                if (isset( $link['tooltiponly'] ) && $link['tooltiponly']) {
+                    $nav[$section][$key]['key'] =
+                        Linker::tooltip( $xmlID );
+                } else {
+                    $nav[$section][$key]['key'] =
+                        Xml::expandAttributes( Linker::tooltipAndAccesskeyAttribs( $xmlID ) );
+                }
+            }
+        }
+        $this->data['namespace_urls'] = $nav['namespaces'];
+        $this->data['view_urls'] = $nav['views'];
+        $this->data['action_urls'] = $nav['actions'];
+        $this->data['variant_urls'] = $nav['variants'];
+
+        // Reverse horizontally rendered navigation elements
+        if ($this->data['rtl']) {
+            $this->data['view_urls'] =
+                array_reverse( $this->data['view_urls'] );
+            $this->data['namespace_urls'] =
+                array_reverse( $this->data['namespace_urls'] );
+            $this->data['personal_urls'] =
+                array_reverse( $this->data['personal_urls'] );
+        }
+
+        if (strpos($_SERVER["SERVER_NAME"], "stage") !== false) {
+            $this->data['login_url'] = "https://loginstage.microfocus.com/nidp/idff/sso?sid=0";
+            $this->data['signup_url'] = "https://secure-wwwstage.novell.com/selfreg/jsp/createOpenSuseAccount.jsp?login=Sign+up";
+        } else {
+            $this->data['login_url'] = "https://login.microfocus.com/nidp/idff/sso?sid=0";
+            $this->data['signup_url'] = "https://secure-www.novell.com/selfreg/jsp/createOpenSuseAccount.jsp?login=Sign+up";
+        }
```

Comment:
Feel free to drop the loginstage part and always use what is the `else` branch now.

(I don't even know how to get an account on loginstage.microfocus.com, so testing with that is impossible for most people.)